src/main/java/frc/robot/Constants2024.java

```diff
@@ -1,16 +1,16 @@
 package frc.robot;
 
 public class Constants2024 extends GameConstants {
-
     //RAMP
-
     public static final double RAMP_PID_P = 1.00;
     public static final double RAMP_PID_I = 0.00;
     public static final double RAMP_PID_D = 0.00;
     public static final double RAMP_PID_FF = 1.00;
     public static final double RAMP_ERROR_IZONE = 0.00;
     public static final double RAMP_POS = 0.0;
     public static final int RAMP_ID = 47;
+    public static final double RAMP_MAX_RPM_VELOCITY = 500;
+    public static final double RAMP_MAX_RPM_ACCELERATION = 1500;
```

Comment:
These look made up, might not hurt to leave a todo somewhere that these are not final or people may take them as truth.